suites/suites/suites.yaml

```diff
@@ -389,8 +391,10 @@ handler_properties:
   aws_ec2:
     ubuntu_trusty_image_id: '{{aws_ec2_eu_central_1_ami_ubuntu_trusty_image_id}}'
     centos_7_image_id: '{{aws_ec2_eu_central_1_ami_centos_7_image_id}}'
+    aws_ec2_eu_west_1_ami_rhel_7_image_id: '{{aws_ec2_eu_west_1_ami_rhel_7_image_id}}'
```

Comment:
Yep, this is actually what I was writing about on flowdoc ;) The name is good for a variable, but we should have one clean name here and rely on specific configs